src/Form/SettingsForm.php

```diff
@@ -74,6 +77,18 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#process_default_value' => FALSE,
     ];
 
+    $form['introduction_text'] = [
+      '#type' => 'text_format',
+      '#title' => $this->t('Introduction text'),
+      '#default_value' => $introduction_text['value'] ?? '',
+      '#allowed_formats' => !empty($formats) ? array_keys($formats) : [],
+      '#description' => $this->t('Text displayed on "My subscriptions" page.'),
+    ];
+
+    if (!empty($introduction_text['format'])) {
+      $form['introduction_text']['#format'] = $introduction_text['format'];
```

Comment:
Why this is not together with the rest? Just ?? NULL,

Reply:
Didn't want to use a null coalescing operator with a null value